tests/test_rbd_csi_default_sc.py

```diff
@@ -0,0 +1,208 @@
+from ocs import pod
+import yaml
+from ocsci.enums import StatusOfTest
+from utility import templating
+from ocs import ocp
+from ocsci import run_this, EcosystemTest, tier1
+import logging
+
+log = logging.getLogger(__name__)
+
+RBD_POOL_YAML = os.path.join("templates/ocs-deployment", "pool.yaml")
+SC_RBD_YAML = os.path.join("templates/ocs-deployment",
+                           "storageclass-csi-rbd.yaml")
+SECRET_RBD_YAML = os.path.join("templates/ocs-deployment", "secret_rbd.yaml")
+PVC_RBD_YAML = os.path.join("templates/ocs-deployment", "pvc-rbd.yaml")
+TEMP_YAML_FILE = 'test_RbdCSI.yaml'
+
+SC = ocp.OCP(
+    kind='StorageClass', namespace="rook-ceph"
+)
+POOL = ocp.OCP(
+    kind='CephBlockPool', namespace="rook-ceph"
+)
+DEPLOYMENT = ocp.OCP(
+    kind='Deployment', namespace="rook-ceph"
+)
+SECRET = ocp.OCP(
+    kind='Secret', namespace="default"
+)
+PVC = ocp.OCP(
+    kind='PersistentVolumeClaim', namespace="rook-ceph"
+)
+
+def create_rbd_pool():
+    """
+    Create Blockpool with default values
+    """
+    file_y = templating.generate_yaml_from_jinja2_template_with_data(
+        RBD_POOL_YAML
+    )
+    with open(TEMP_YAML_FILE, 'w') as yaml_file:
+        yaml.dump(file_y, yaml_file, default_flow_style=False)
+    log.info(f"Creating a new CephBlockPool with default name")
+    assert POOL.create(yaml_file=TEMP_YAML_FILE)
+    sleep(5)
+    log.info(f"Sleeping for 5 sec to let pool get created")
+    return True
+
+def create_storageclass_rbd():
+    """
+    This Creates a default CSI StorageClass
+    """
+    file_y = templating.generate_yaml_from_jinja2_template_with_data(
+        SC_RBD_YAML
+    )
+    with open(TEMP_YAML_FILE, 'w') as yaml_file:
+        yaml.dump(file_y, yaml_file, default_flow_style=False)
+    log.info(f"Creating a RBD StorageClass with default values")
+    assert SC.create(yaml_file=TEMP_YAML_FILE)
+    log.info(f"Sleeping for 5 sec to let StorageClass get created")
+    sleep(2)
+    return True
+
+def validate_pool_creation(pool_name):
+    """
+    Check whether default blockpool is created or not at ceph and as well
+    OCS side
+
+    :param pool_name:
+    :return:
+    """
+    ceph_validate = False
+    OCS_validate = False
+    cmd = "ceph osd lspools|grep " + str(pool_name) + " |awk '{print$2}'"
+    took_box = get_toolbox_pod_handler()
+    out, err, ret = took_box.exec_command(cmd=cmd, timeout=20)
+    if out:
+        log.info(f"{pool_name} pool is created successfully at CEPH side")
+        ceph_validate = True
+    else:
+        log.error(f"{pool_name} pool failed to get created at CEPH side")
+    pool_obj = (POOL.get(resource_name=""))
+    sample = pool_obj['items']
+    for item in sample:
+        if item.metadata.name == pool_name:
+            OCS_validate = True
+    if ceph_validate and OCS_validate:
+        log.info("Pool got created successfully from Ceph and OCS side")
+        return True
+    else:
+        return False
+
+def get_toolbox_pod_handler():
```

Comment:
No need as we already have this https://github.com/red-hat-storage/ocs-ci/blob/master/ocs/ocp.py#L252